guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,9 +16,6 @@ endif::[]
 For more information, see {AdministeringDocURL}backing-up-{project-context}-server-and-{smart-proxy-context}_admin[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
 * Plan for updating any scripts you use that contain {Project} API commands because some API commands differ between versions of {Project}.
 ifdef::satellite[]
-* Migrate all organizations to Simple Content Access (SCA).
-For more information, see the Red Hat Knowledgebase solution https://access.redhat.com/articles/simple-content-access[Simple Content Access]. 
-
```

Comment:
Before:
![Screenshot From 2025-01-15 10-25-11](https://github.com/user-attachments/assets/b3438c02-3b97-44ee-91cd-efca7363d42c)
After:
![Screenshot From 2025-01-15 10-25-25](https://github.com/user-attachments/assets/31af7341-2b3e-44a3-aa26-09c5bf21a916)

Note how the original line 22 is now part of the preceding bullet point. I don't think that was intended. Re-adding a blank line (21) should take care of it.

Reply:
Good catch. Alternative: make it another bullet point.